cli.js

```diff
@@ -114,7 +114,16 @@ ee.on('finish', async () => {
 	ora.text = 'Code signing DMG';
 
 	try {
-		await execa('codesign', ['--sign', 'Developer ID Application', dmgPath]);
+		const {stdout} = await execa('security', ['find-identity', '-v', '-p', 'codesigning'])
+		if (/Developer ID Application/m.exec(stdout)) {
```

Comment:
Could just be `if (stdout.includes('Developer ID Application')) {`. We should probably do this just to be safe: `if (stdout.includes('"Developer ID Application:')) {`